### PR TITLE
[PHPSTAN] Fix: Wrong binary operations

### PR DESCRIPTION
**Pull request by blankse**
Fix following phpstan level 2 errors:
```
  Line   bundles/EcommerceFrameworkBundle/IndexService/ProductList/DefaultFactFinder.php
  787    Binary operation "." between string and array<string> results in an error.
  
  Line   bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php
  171    Binary operation "*" between string and 100 results in an error.
  198    Binary operation "*" between string and 100 results in an error.

  Line   lib/helper-functions.php
  269    Binary operation "*=" between string and 1024 results in an error.

  Line   models/Asset/Image/Thumbnail.php                                                                                        
  246    Binary operation "." between '© ' and array|null results in an error.                                            
  247    Binary operation "." between '© ' and array|null results in an error.                                            
  
  Line   models/DataObject/ClassDefinition/Data/Localizedfields.php                                                                                      
  1262   Binary operation "&" between Pimcore\Model\DataObject\ClassDefinition\Data|null and bool results in an error.                            

  Line   models/Document/Tag/Date.php                                                                                                    
  167    Binary operation "." between 'cannot get document…' and array<object> results in an error.                               
 
  Line   models/Document/Tag/Video.php                                                                                                       
  436    Binary operation "-" between string and 1 results in an error.                                                               
```

### CODE REVIEW

File: bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php [hunk 1]
```diff
@@ -195,7 +195,7 @@ protected function setProducts(array $items): array
             $product = $item->getProduct();
 
             $products[$key]['name'] = $product->getName();
-            $products[$key]['unitPrice'] = (string) round($product->getOSPrice()->getAmount()->asNumeric(), 2) * 100;
+            $products[$key]['unitPrice'] = round($product->getOSPrice()->getAmount()->asNumeric(), 2) * 100;
```
Comment (brusch) [inline bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php]: It seems that PayU expects this as a string ... same for `totalAmount` , see: 
http://developers.payu.com/en/restapi.html#order_data_retrieve

Comment (brusch) [reply bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php]: Fixed it already 😉

**Comment by brusch** [comment]
Thanks!

File: bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php (hunk 1 above)
Comment (blankse) [reply bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php]: Yes, than it have to be in brackets 👍

**Pull request merged**